unfoldingWord/tc-create-app
# Use SIL webfonts for translation

https://github.com/unfoldingword/tc-create-app/issues/1411
Requires changes from markdown-translatable and datatable-translatable to show the selected font.
https://github.com/unfoldingWord/datatable-translatable/pull/117
https://github.com/unfoldingWord/markdown-translatable/pull/93

Ready for Review.
Test Instructions: 
1. Checkout Branch `feature-superdav42-use-webfonts` in tcCreate. And in datatable-translatable and markdown translatable. Use yalc publish in datatable-translatable and markdown translatable. And in tcCreate use `yalc link datatable-translatable` and `yalc link markdown-translatable`
1. Open TSV file.
2. Change font in sidebar. 
3. Font should immediately change.
1. Open md file.
2. Change font in sidebar. 
3. Font should immediately change.
4. Greek should still be in Ezra font. It is now using a newer version served from the app instead of third party.
5. Warning should show when using a graphite font and not using firefox.
